### ISSUE TITLE
I propose a change to the beginning of "Regular expressions" and "Patterns and flags" pages

### ISSUE THREAD

**Issue by luc4leone** [javascript-tutorial/en.javascript.info#962]
The "Regular expressions" page begins this way:

<img src="https://user-images.githubusercontent.com/2797254/57223890-df99ee80-7007-11e9-9f9c-7e0d5eef7fb3.png" width="540" />

immediately followed by the "Patterns and flags" page, which begins this way:

<img src="https://user-images.githubusercontent.com/2797254/57223947-107a2380-7008-11e9-9183-854b15e35b04.png" width="540" />

I propose to delete that repetition.

Also, the "Patterns and flags" page continue like this:

<img src="https://user-images.githubusercontent.com/2797254/57224020-52a36500-7008-11e9-90c3-4852bc366866.png" width="540" />

This part

>In JavaScript regular expressions are implemented using objects of a built-in RegExp class and integrated with strings.
>
>Please note that regular expressions vary between programming languages. In this tutorial we concentrate on JavaScript. Of course there’s a lot in common, but they are a somewhat different in Perl, Ruby, PHP etc.

according to me, belongs to the "Regular expressions" page, and I propose to move it there. I also propose to delete the "Regular expression" heading, it's a repetition and it's confusing in the table of content in the left sidebar:

<img src="https://user-images.githubusercontent.com/2797254/57224199-f42ab680-7008-11e9-9d50-b45e51b15684.png" width="180" />.

To summarize, my proposal is to split the following content over the 2 pages accordingly:

## Regular expressions

Regular expressions is a powerful way of doing search and replace in strings.

In JavaScript regular expressions are implemented using objects of a built-in RegExp class and integrated with strings.

Please note that regular expressions vary between programming languages. In this tutorial we concentrate on JavaScript. Of course there’s a lot in common, but they are a somewhat different in Perl, Ruby, PHP etc.

## Patterns and flags

A regular expression (also “regexp”, or just “reg”) consists of a pattern and optional flags.

There are two syntaxes to create a regular expression object.

**Comment by iliakan** [comment]
Could you please make this as a pull request?

**Comment by luc4leone** [comment]
>Could you please make this as a pull request?

Sure 👍

**Comment by iliakan** [comment]
Thank you!

**Comment by luc4leone** [comment]
Thank you for the tutorial!
